schema/swagger-extensions.json

```diff
@@ -1589,6 +1592,51 @@
         "enum": ["create", "read", "update"]
       },
       "uniqueItems": true
+    },
+    "xmsResponseExample": {
+      "type": "object",
+      "description": "Provides the format of specifying an example response in the x-ms-examples extension.",
+      "properties": {
+        "headers": {
+          "type": "object",
+          "description": "Examples of response headers like 'location', 'azure-asyncoperation', 'retry-after', etc.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "body":{
+          "type": ["object", "number", "array", "integer", "string", "boolean", "null"],
```

Comment:
Could perhaps use further description - If 'null' is specified or 'string' is specified, I wonder how they are interpreted? An empty body, or a body containing the text 'null' (which I have seen some servers return :p) ? Would a string value be assumed to be plain text of the body?